lbrynet/analytics/track.py

```diff
@@ -0,0 +1,24 @@
+import collections
+
+
+class Track(object):
+    """Track and summarize observations of metrics."""
+    def __init__(self):
+        self.data = collections.defaultdict(list)
+
+    def add_observation(self, metric, value):
+        self.data[metric].append(value)
+
+    def summarize(self, metric, op=sum):
```

Comment:
i would call this something like `summarize_and_reset` so it's clear it has unexpected side effects. or i'd make reset a separate method